orchagent/crmorch.cpp

```diff
@@ -504,22 +533,36 @@ void CrmOrch::updateCrmCountersTable()
     // Update CRM used counters in COUNTERS_DB
     for (const auto &i : crmUsedCntsTableMap)
     {
-        for (const auto &cnt : m_resourcesMap.at(i.second).countersMap)
+        try
+        {
+            for (const auto &cnt : m_resourcesMap.at(i.second).countersMap)
```

Comment:
Since you are erasing the unsupported resource, do we hit this case?

Reply:
The code iterates over the **constant** map crmUsedCntsTableMap which includes the unsupported resources as well. The out_of_range exception will be hit when it tries to access m_resourcesMap.at(i.second).countersMap for any of the unsupported resources.